salt/modules/acme.py

```diff
@@ -133,6 +135,8 @@ def cert(name,
                          the port Certbot listens on. A conforming ACME server
                          will still attempt to connect on port 80.
     :param https_01_address: The address the server listens to during http-01 challenge.
+    :param dns_plugin: Name of a DNS plugin to use
```

Comment:
We need documentation somewhere on possible values for this or at least some examples. Otherwise, nobody will know how to use this. :) Same for the plugin credentials.